core/templates/services/context.service.spec.ts

```diff
@@ -64,6 +64,24 @@ describe('Context service', () => {
       expect(ecs.getQuestionPlayerIsManuallySet()).toEqual(false);
       expect(ecs.isInQuestionPlayerMode()).toEqual(false);
     });
+
+    it('should affirm the page context to be exploration player or editor',
+      ()=> {
+        expect(ecs.isInExplorationContext()).toBe(true);
+      });
+
+    it('should correctly set and retrive the exploration id', () => {
+      ecs.setExplorationId('1000');
+      expect(ecs.getExplorationId()).toBe('1000');
+    });
+
+    it('should correctly affirm that page contains reference to skills',
```

Comment:
Here and below please make the test description more specific please. Avoid having duplicate descriptions.

Reply:
Done!